.github/actions/tests/Dockerfile

```
@@ -7,7 +7,8 @@ LABEL \
 
 USER 0
 
-RUN dnf install -y python-devel fedora-packager gcc krb5-devel openldap-devel gobject-introspection-devel cairo-gobject-devel
+RUN dnf install -y python-devel fedora-packager gcc krb5-devel openldap-devel gobject-introspection-devel cairo-gobject-devel tox
+RUN sed -i 's/test-requirements/yum-packages.txt/g' tox.ini
```

These files are not interchangeable: `test-requirements.txt` contains PyPI packages, `yum-packages.txt` Fedora package.

Better way could be to empty the file instead and install packages from the `yum-*.txt` directly:

```suggestion
RUN dnf install -y tox $(sed 's/#.*//' yum-packages.txt yum-packages-devel.txt)
RUN echo > test-requirements.txt
```